tests/rule_based_profiler/parameter_builder/test_mean_unexpected_map_metric_multi_batch_parameter_builder.py

```diff
@@ -275,3 +277,47 @@ def test_mean_unexpected_map_metric_multi_batch_parameter_builder_bobby_datetime
         atol=atol,
         err_msg=f"Actual value of {actual_parameter_value.value} differs from expected value of {expected_parameter_value} by more than {atol + rtol * abs(actual_parameter_value.value)} tolerance.",
     )
+
+
+def test_mean_unexpected_map_metric_multi_batch_parameter_builder_bobby_serialization(
```

Comment:
actually could we make the test name a bit more specific / extend the test? The thing we are testing at the end are that the keys.  If it could be something like `_check_serialized_keys`, or have it do the full serialization-deserialization loop to ensure the steps are working correctly